bazel/python_packages.bzl

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python repository"""
```

Comment:
I am a little confused by the file names between files `bazel/python_packages.bzl` (this file), and `bazel/python_repositories.bzl` and `bazel/python_toolchain.bzl`. The `python_toolchain.bzl` file seems named reasonably.  But why is this file (`python_packages.bzl`) named such when it is ~entirely about establishing / installing a python interpreter of a specified version? Sounds a lot like `python_toolchain.bzl`? Further there is this comment at line 12 of this file `Python repository` which seems mismatched with the name of this file (relative to the other file which is in fact named `python_repository.bzl`).  Long story short I'm simply a little confused by the file naming and wonder if it could be improved (or contents re-arranged). Thoughts?

Reply:
The idea behind the files names and the subdivision of the content is to clarify the distinction between external python repos, Python interpreter packages, and the Python toolchain definition used in Bazel.

`bazel/python_repositories.bzl` contains all external repositories relevant for Python, e.g. Subpar.    

`bazel/python_packages.bzl` includes the Python 2 and 3 Interpreter packages that are installed when used as part of the toolchain. Currently, there exists only the Python 3 interpreter, but the file could be extended with a Python 2 version.
 
`python_toolchain.bzl` describes the combination of available Python interpreters that forms a toolchain. These Python interpreters could be defined in `python_packages` as well as pre-installed in the underlying system.

Reply:
The comment in line 12 is misleading. I changed it.